rules.d/60-block.rules

```diff
@@ -11,3 +11,6 @@ ACTION=="change", SUBSYSTEM=="scsi", ENV{DEVTYPE}=="scsi_device", TEST=="block",
 ACTION!="remove", SUBSYSTEM=="block", \
   KERNEL=="loop*|mmcblk*[0-9]|msblk*[0-9]|mspblk*[0-9]|nvme*|sd*|vd*|xvd*|bcache*|cciss*|dasd*|ubd*|ubi*|scm*|pmem*|nbd*|zd*", \
   OPTIONS+="watch"
+
```

Comment:
this should read the iocst params into a device property as first step. And then, if that property is actually set we should via a second rule apply the data.

i.e. the hwdb hookup should be done via the normal udev hwdb builtin. the new iocost component should not bother with hwdb itself.

Reply:
This is done but moved to its own file, let me know if you'd prefer it on 60-block.rules.